drivers/interrupt_controller/intc_plic.c

```diff
@@ -29,13 +29,39 @@
 #define PLIC_IRQS        (CONFIG_NUM_IRQS - CONFIG_2ND_LVL_ISR_TBL_OFFSET)
 #define PLIC_EN_SIZE     ((PLIC_IRQS >> 5) + 1)
 
+#define EDGE_TRIG_TYPE   (PLIC_MAX_PRIO + 0x1080)
```

Comment:
and maybe some reference for the `0x1080`

Reply:
Yeah, I was also curious about that offset.